engine/apps/heartbeat/models.py

```diff
@@ -48,16 +48,6 @@ class IntegrationHeartBeat(models.Model):
     Stores the latest received heartbeat signal time
     """
 
-    last_checkup_task_time = models.DateTimeField(default=None, null=True)
-    """
-    Deprecated. This field is not used. TODO: remove it
-    """
-
-    actual_check_up_task_id = models.CharField(max_length=100)
-    """
-    Deprecated. Stored the latest scheduled `integration_heartbeat_checkup` task id. TODO: remove it
-    """
```

Comment:
while testing integration heartbeat functionality saw these TODOs and decided to clean them up